app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee

```diff
@@ -1146,6 +1147,10 @@ ShortFormApplicationService = (
       ListingDataService.listing.Custom_Listing_Type
     )
 
+  # TODO: The Kelsey - find a better way to ID before this goes live
+  Service.listingHasHomeAndCommunityBasedServicesUnits  = (listing) ->
```

Comment:
What about the adding a type to the custom listing type field? That field is currently only used for educator listings

Reply:
That would require additional Salesforce work, and an additional instruction for the person setting up the listing in prod. It is one potential solution, but not in scope of this PR.